RecoLocalTracker/SiPixelRecHits/src/PixelCPEClusterRepair.cc

```diff
@@ -156,8 +156,6 @@ void PixelCPEClusterRepair::fill2DTemplIDs() {
 //  Clean up.
 //-----------------------------------------------------------------------------
 PixelCPEClusterRepair::~PixelCPEClusterRepair() {
-  for (auto x : thePixelTemp_)
-    x.destroy();
   for (auto x : thePixelTemp2D_)
```

Comment:
Couldn't the `destroy` for 2D template be removed as well?

Reply:
Only if I went and changed that class as well. I was hoping to limit this pull request to just the one class that has the problem.

Reply:
Is the signing of this pull request being held up by this?

Reply:
> Is the signing of this pull request being held up by this?

No @Dr15Jones , mine was only an observation I made while quickly scrolling through this PR during our periodic reco chat.
I still believe that some symmetry between the two rather similar 1D and 2D templates should be better maintained, but I'll let Joosep decide about it, who's going to review this PR for reco

Reply:
it's a good point that it should be cleaned up, I opened an issue https://github.com/cms-sw/cmssw/issues/32567